Cargo.toml

```diff
@@ -10,4 +10,4 @@ bitcoin = "0.29.2"
 futures = "0.3.26"
 lightning = "0.0.114"
 tokio = { version = "1.25.0", features = ["rt", "rt-multi-thread"] }
-tonic_lnd = { git = "https://github.com/Kixunil/tonic_lnd", rev = "c30b614" } 
+tonic_lnd = { version = "0.5.1", git = "https://github.com/Kixunil/tonic_lnd"} 
```

Comment:
I think that this could just be `tonic_lnd = "0.5.1"` now that we don't specifically need to point to a commit?

Reply:
Sure, sounds good.

Reply:
You can remove this first commit now?

Reply:
I deleted not.
Thanks